docs/source/faq/general.md

```diff
@@ -31,6 +31,44 @@ Manim will not call your method and simply output a black frame.
 
 ---
 
+## What are the default measurements for Manim's scene?
+
+The scene measures 8 units in height and has a default ratio of 16:9,
+which means that it measures :math:`8 \cdot 16 / 9 = 14 + 2/9` units in width.
```

Comment:
This doesn't render for some reason: https://manimce--2767.org.readthedocs.build/en/2767/faq/general.html#what-are-the-default-measurements-for-manim-s-scene

Reply:
I forgot that I am writing markdown, not ReST.

```suggestion
which means that it measures {math}`8 \cdot 16 / 9 = 14 + 2/9` units in width.
```